.changeset/tidy-radios-watch.md

```diff
@@ -0,0 +1,66 @@
+---
+"@salt-ds/highcharts-theme": patch
+---
+
+### Summary
+
+Salt Highcharts theme is a package that supplies the Salt theme for Highcharts. It includes a CSS file for all CSS-based styles, and an options object for styles that are not CSS configurable. Together, they provide a complete theming solution for Highcharts version 10 to version 12.
```

Comment:
the options object is a WIP, not sure what it will contain until we get through all the charts.
Will need to understand the choices of size/spacing, formatting, and the impact of density.
Might flag that up, so folks are aware this object is not final